interactions/models/discord/enums.py

```diff
@@ -611,6 +611,8 @@ class ChannelType(CursedIntEnum):
     """Voice channel for hosting events with an audience"""
     GUILD_FORUM = 15
     """A Forum channel"""
+    MEDIA_CHANNEL = 16
```

Comment:
Addressed likewise with [`channel.py`](https://github.com/interactions-py/interactions.py/pull/1458/files#r1248970595)'s definition:

```suggestion
    GUILD_MEDIA = 16
```